packages/components/src/internal/components/navigation/UserMenuGroup.tsx

```diff
@@ -120,7 +120,7 @@ export const UserMenuGroupImpl: FC<UserMenuProps & ImplProps> = props => {
                     <div className="navbar-connector" />
                     {userMenuItems}
                     {extraUserItems}
-                    <MenuDivider />
+                    {(userMenuItems.length > 0 || extraUserItems) && <MenuDivider />}
```

Comment:
can userMenuItems be undefined at any point?

Reply:
I don't think so, but I'll be defensive anyway.